bc-react/src/stores/UserStore.js

```diff
@@ -36,6 +35,12 @@ class UserStore extends EventEmitter{
         this.emit('login-fail')
         break
       }
+      case("DELETE_USER"):{
```

Comment:
you delete them in atom and push that change up to this branch, or if you have other code checkout out you can use the inline atom editor to do it